backend/src/game/newcards/discard_cards/victory_point_manipulation_cards/test_evacuation_orders.py

```diff
@@ -0,0 +1,14 @@
+from backend.src.game.newcards.discard_cards.victory_point_manipulation_cards.evacuation_orders import \
+    EvacuationOrders
+
+
+def test_evacuation_orders_subtracts_5_victory_points(five_players):
+    for player in five_players:
+        player.update_victory_points_by(10)
+    EvacuationOrders().immediate_effect(None, five_players)
+    assert all(other_players.victory_points ==
+               5 for other_players in five_players)
```

Comment:
Would it be good to have a test case that asserts that VP doesn't go below 0? Might not be this classes' responsibility, but that test needs to exist somewhere.